### DISCUSSION TITLE
Last slide

### DISCUSSION THREAD

**Discussion by kensyone** [blockscout/blockscout#11303] [Ideas]
Lets add “Last” slide too

![IMG_2902](https://github.com/user-attachments/assets/7f92bc98-c0f9-4426-a4ae-d452df757361)

**Comment by kensyone** [comment]
blockscan has the feature

![IMG_2903](https://github.com/user-attachments/assets/59050185-0b5e-41f1-9b85-7fab5709bbff)

**Comment by sl1depengwyn** [comment]
Do you think adding a possibility to reverse order token transfer would fulfil your use case? I.e. you would be able to sort by block number to see the earliest token transfers, or the last page in terms of etherscan
